src/main/java/com/example/garage/service/CarService.java

```
@@ -0,0 +1,61 @@
+package com.example.garage.service;
+
+import com.example.garage.model.Car;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Getter
+@Setter
+public class CarService {
+
+    public static List<Car> cars = new ArrayList<>();
+
+    @Autowired
+    public CarService(List<Car> cars) {
+        CarService.cars = cars;
+    }
+
+    int carCounter = 1;
+
+    public Car createCar(Car car) {
+        car.setId(carCounter++);
+        cars.add(car);
+        return car;
+    }
+
+    public Car getCar(int id) {
+        for (Car car : cars) {
+            if (car.getId() == id)
+                return car;
+        }
+        return null; // ???
```

if the car not found throw an exception instead of null